bedrock/firefox/templates/firefox/includes/family-nav.html

```diff
@@ -0,0 +1,153 @@
+          <li><a href="{{ url('firefox.desktop.customize') }}" data-id="desktop-customize">{{ _('Flexible') }}</a></li>
+          <li><a href="{{ url('firefox.desktop.fast') }}" data-id="desktop-fast">{{ _('Fast') }}</a></li>
+        </ul>
+
+        <ul class="subnav" data-parent="android">
+          <li><a href="{{ url('firefox.android.index') }}" data-id="android-index">{{ _('Overview') }}</a></li>
+        </ul>
+
+        <ul class="subnav active" data-parent="os">
+          <li><a href="{{ url('firefox.os.index') }}" data-id="os-index">{{ _('About') }}</a></li>
+          <li><a href="{{ url('firefox.os.devices') }}" data-id="os-devices">{{ _('Devices') }}</a></li>
+          <li><a href="{{ url('firefox.partners.index') }}" data-id="os-partners">{{ _('For partners') }}</a></li>
+          <li><a href="{{ url('firefox.os.mwc-2015-preview') }}" data-id="os-mwc"><i class="icon-mwc"></i>{{ _('MWC 2015') }}</a></li>
+        </ul>
+      </div> {# --/#fxfamilynav-subnav --#}
+
+      <div id="fxfamilynav-tertiarynav">
+        {# the triangle needs a border, so :before content doesn't cut it #}
+        <div class="triangle"></div>
+        <div class="triangle-border"></div>
+
+        <ul class="tertiarynav" data-parent="desktop">
+          <li><a rel="external" href="https://support.mozilla.org/products/firefox">{{ _('Support') }}</a></li>
+          <li><a href="{{ url('firefox.dnt') }}">{{ _('DNT') }}</a></li>
+          <li><a rel="external" href="https://support.mozilla.org/kb/private-browsing-browse-web-without-saving-info">{{ _('Private Browsing') }}</a></li>
+          <li><a rel="external" href="https://addons.mozilla.org">{{ _('Add-Ons') }}</a></li>
+        </ul>
+
+        <ul class="tertiarynav" data-parent="android">
+          <li><a rel="external" href="https://support.mozilla.org/products/mobile">{{ _('Support') }}</a></li>
+          <li><a href="{{ url('firefox.dnt') }}">{{ _('DNT') }}</a></li>
+          <li><a rel="external" href="https://www.youtube.com/watch?v=Ewk0YtFTBec&amp;list=PLA8ABF6B7A66A0C5E&amp;index=15">{{ _('Private Browsing') }}</a></li>
+          <li><a rel="external" href="https://addons.mozilla.org/android/">{{ _('Add-Ons') }}</a></li>
+        </ul>
+
+        <ul class="tertiarynav" data-parent="os">
+          <li><a rel="external" href="https://support.mozilla.org/products/firefox-os">{{ _('Support') }}</a></li>
+          <li><a rel="external" href="https://blog.mozilla.org/blog/category/firefox-os/">{{ _('News') }}</a></li>
+          <li><a rel="external" href="https://marketplace.firefox.com/">{{ _('Marketplace') }}</a></li>
+          <li><a rel="external" href="https://developer.mozilla.org/Firefox_OS">{{ _('Developers') }}</a></li>
+        </ul>
+      </div> {#-- /#fxfamilynav-tertiarynav --#}
+    </nav> {#-- /#fxfamillynav --#}
+
+    {# Page will dictate CTA (button, usually) to be placed in this wrapper #}
+    <div id="fxfamilynav-cta-wrapper"></div>
```

Comment:
Could this be done via a conditional {% block %} instead of injected using JS maybe? Just a thought for the near future, no need to change anything for this PR :smile: